java/ql/src/semmle/code/java/frameworks/play/PlayAddCSRFToken.qll

```diff
@@ -0,0 +1,13 @@
+import java
+
+/**
+ * Play Framework AddCSRFToken
+ *
+ * @description Gets the methods using AddCSRFToken annotation.
+ * (https://www.playframework.com/documentation/2.6.x/JavaBodyParsers#Choosing-an-explicit-body-parser)
+ */
+class PlayAddCSRFTokenAnnotation extends Annotation {
```

Comment:
Unused class?

Reply:
@smowton these were classes added if somebody wants to use them. As an example: we use them at Linkedin to sort of find maybe a method/resource with the csrf-annotation for play configuration not set globally to have CSRF or to override it. This is the similar reasoning taken for some of the classes which you mentioned being unused.

Reply:
OK, just checking if there was some missing piece of the PR that hadn't been `git add`'d